zapcore/entry.go

```diff
@@ -104,67 +104,70 @@ type Entry struct {
 	Stack      string
 }
 
-// CheckWriteAction indicates what action to take after (*CheckedEntry).Write
-// is done. Actions are ordered in increasing severity.
+// CheckWriteAction indicates what action to take after Write is done. Actions
+// are ordered in increasing severity.
 type CheckWriteAction uint8
 
 const (
-	// WriteThenNoop is the default behavior to do nothing speccial after write.
+	// WriteThenNoop indicates that nothing special needs to be done. It's the
+	// default behavior.
 	WriteThenNoop = CheckWriteAction(iota)
```

Comment:
While this works fine, the standard way to define constants of a custom type is `WriteThenNoop CheckWriteAction = iota`.

See examples,
https://golang.org/pkg/time/#Weekday
https://golang.org/pkg/reflect/#ChanDir
https://golang.org/pkg/reflect/#Kind

Since this shows up in documentation, I'd prefer if we used the standard syntax to avoid any confusion.